README.md

```diff
@@ -3,3 +3,91 @@
 HyperUI is a collection of free Tailwind CSS components that can be used in your next project. With a range of components, you can build your next marketing website, admin dashboard, eCommerce store and much more.
 
 ![](https://hyperui.dev/og.jpg)
+
+## Table of Contents
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+- [Components](#components)
+    - [🤖 Application UI](#application-ui)
+    - [📣 Marketing](#marketing)
+    - [🛍️ eCommerce](#ecommerce)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+Hyper UI doesn't require any installation. All you need to do is browse components you need to implement in your project, copy its code, and simply paste it to your project.
```

Comment:
You could grab the information from the FAQs for this:

> There is no HyperUI installation, if you have a Tailwind CSS project, you can copy the code and paste it into your project. Some components do use Alpine JS.

Does that make sense, or do you think it should be reworded?

Reply:
I totally missed the FAQs. This looks good @markmead 👍